module/dice/dice.mjs

```diff
@@ -51,7 +52,7 @@
 export async function d20Roll({
   parts=[], data={}, event,
   advantage, disadvantage, critical=20, fumble=1, targetValue,
-  elvenAccuracy, halflingLucky, reliableTalent,
+  elvenAccuracy, halflingLucky, reliableTalent, formulaOnly,
```

Comment:
Adding a new function to the d20Roll function. Instead of always rolling a dice, you can ask for something akin to a skill check etc, and if passed a true value, instead of rolling a dice, it will make the formula then return that formula. This is used in my updated initiative function where it uses this to generate the formula then pass that to the roll initiative function.